docker/GPU/Dockerfile

```diff
@@ -0,0 +1,85 @@
+FROM nvcr.io/nvidia/tensorrt:21.04-py3
+
+ARG CUDA=10.2
+ARG PYTHON_VERSION=3.8
+ARG TORCH_VERSION=1.8.0
+ARG TORCHVISION_VERSION=0.9.0
+ARG ONNXRUNTIME_VERSION=1.8.1
+ARG MMCV_VERSION=1.4.0
+ARG CMAKE_VERSION=3.20.0
+ENV FORCE_CUDA="1"
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+### update apt and install libs
+RUN apt-get update &&\
+    apt-get install -y vim libsm6 libxext6 libxrender-dev libgl1-mesa-glx git wget libssl-dev libopencv-dev libspdlog-dev --no-install-recommends &&\
+    rm -rf /var/lib/apt/lists/*
+
+RUN curl -fsSL -v -o ~/miniconda.sh -O  https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh  && \
+    chmod +x ~/miniconda.sh && \
+    ~/miniconda.sh -b -p /opt/conda && \
+    rm ~/miniconda.sh && \
+    /opt/conda/bin/conda install -y python=${PYTHON_VERSION} conda-build pyyaml numpy ipython cython typing typing_extensions mkl mkl-include ninja && \
+    /opt/conda/bin/conda clean -ya
+
+### pytorch
+RUN /opt/conda/bin/conda install pytorch==${TORCH_VERSION} torchvision==${TORCHVISION_VERSION} cudatoolkit=${CUDA} -c pytorch
+ENV PATH /opt/conda/bin:$PATH
+
+### install mmcv-full
+RUN /opt/conda/bin/pip install mmcv-full==${MMCV_VERSION} -f https://download.openmmlab.com/mmcv/dist/cu${CUDA//./}/torch${TORCH_VERSION}/index.html
+
+WORKDIR /root/workspace
+### get onnxruntime
+RUN wget https://github.com/microsoft/onnxruntime/releases/download/v${ONNXRUNTIME_VERSION}/onnxruntime-linux-x64-${ONNXRUNTIME_VERSION}.tgz \
+    && tar -zxvf onnxruntime-linux-x64-${ONNXRUNTIME_VERSION}.tgz &&\
+    pip install onnxruntime-gpu==${ONNXRUNTIME_VERSION}
+
+### cp trt from pip to conda
+RUN cp -r /usr/local/lib/python${PYTHON_VERSION}/dist-packages/tensorrt* /opt/conda/lib/python${PYTHON_VERSION}/site-packages/
+
+### update cmake
+RUN  wget https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/cmake-${CMAKE_VERSION}.tar.gz &&\
+    tar -zxvf cmake-${CMAKE_VERSION}.tar.gz &&\
+    cd cmake-${CMAKE_VERSION} &&\
+    ./bootstrap &&\
+    make &&\
+    make install
+
+### install mmdeploy
+ENV ONNXRUNTIME_DIR=/root/workspace/onnxruntime-linux-x64-${ONNXRUNTIME_VERSION}
+ENV TENSORRT_DIR=/workspace/tensorrt
+ARG VERSION
+RUN git clone https://github.com/open-mmlab/mmdeploy &&\
+    cd mmdeploy &&\
+    if [ -z ${VERSION} ] ; then echo "No MMDeploy version passed in, building on master" ; else git checkout tags/v${VERSION} -b tag_v${VERSION} ; fi &&\
+    git submodule update --init --recursive &&\
+    rm -rf build &&\
+    mkdir build &&\
+    cd build &&\
+    cmake -DMMDEPLOY_TARGET_BACKENDS=ort .. &&\
+    make -j$(nproc) &&\
+    cmake -DMMDEPLOY_TARGET_BACKENDS=trt .. &&\
+    make -j$(nproc) &&\
+    cd .. &&\
+    pip install -e .
+
+### build sdk
+RUN git clone https://github.com/openppl-public/ppl.cv.git &&\
```

Comment:
should install pplnn at first.

Reply:
Seems no need to to that as pplcv is independent from pplnn